engine/api/src/main/scala/pl/touk/nussknacker/engine/api/typed/typing.scala

```diff
@@ -67,6 +70,7 @@ object typing {
 
     override def display: String = s"Dict(id=$dictId)"
 
+    override def javaClassName: String = ???
```

Comment:
Missing implementation? Maybe this method should return: Option[String] and there shuld be just null?

Reply:
true, I was going to change it but I wasn't sure how, thanks

Reply:
Changed the logic there, turns out those implementations weren't necessary I only needed that method in TypedClass

Reply:
Please push the change because I still see those ???

Reply:
Apologies, must have missed it. Should be ok now